File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/TransMerchantInfo.java
package com.ctrip.ccard.creditcard.vcc.bean;

/**
 * Description:
 */
public class TransMerchantInfo {
    /**
     * 商户名称
     */
    private String merchantName;

    /**
     * 商户MCC
     */
    private String merchantCategoryCode;

    /**
     * 商户所在国家码
     */
    private String merchantCountryCode;

    /**
     * 商户所在国家城市
     */
    private String merchantCity;

    /**
     * 商户所在国家城市区域
     */
    private String merchantState;

    /**
     * 商户所在国家邮编
     */
    private String merchantZipCode;

    /**
     * 商户描述
     */
    private String merchantDesc;

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public String getMerchantCategoryCode() {
        return merchantCategoryCode;
    }

    public void setMerchantCategoryCode(String merchantCategoryCode) {
        this.merchantCategoryCode = merchantCategoryCode;
    }

    public String getMerchantCountryCode() {
        return merchantCountryCode;
    }

    public void setMerchantCountryCode(String merchantCountryCode) {
        this.merchantCountryCode = merchantCountryCode;
    }

    public String getMerchantCity() {
        return merchantCity;
    }

    public void setMerchantCity(String merchantCity) {
        this.merchantCity = merchantCity;
    }

    public String getMerchantState() {
        return merchantState;
    }

    public void setMerchantState(String merchantState) {
        this.merchantState = merchantState;
    }

    public String getMerchantZipCode() {
        return merchantZipCode;
    }

    public void setMerchantZipCode(String merchantZipCode) {
        this.merchantZipCode = merchantZipCode;
    }

    public String getMerchantDesc() {
        return merchantDesc;
    }

    public void setMerchantDesc(String merchantDesc) {
        this.merchantDesc = merchantDesc;
    }

}

File: php-sdk/src/triplink/vcc/card/client/query/TransMerchantInfo.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/4
 * Time: 14:35
 */

namespace triplink\vcc\card\client\query;

class TransMerchantInfo
{
    /**
     * 商户名称
     */
    var $merchantName;

    /**
     * 商户MCC
     */
    var $merchantCategoryCode;

    /**
     * 商户所在国家码
     */
    var $merchantCountryCode;

    /**
     * 商户所在国家城市
     */
    var $merchantCity;

    /**
     * 商户所在国家城市区域
     */
    var $merchantState;

    /**
     * 商户所在国家邮编
     */
    var $merchantZipCode;

    /**
     * 商户描述
     */
    var $merchantDesc;

    /**
     * @return mixed
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @param mixed $merchantName
     */
    public function setMerchantName($merchantName): void
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @return mixed
     */
    public function getMerchantCategoryCode()
    {
        return $this->merchantCategoryCode;
    }

    /**
     * @param mixed $merchantCategoryCode
     */
    public function setMerchantCategoryCode($merchantCategoryCode): void
    {
        $this->merchantCategoryCode = $merchantCategoryCode;
    }

    /**
     * @return mixed
     */
    public function getMerchantCountryCode()
    {
        return $this->merchantCountryCode;
    }

    /**
     * @param mixed $merchantCountryCode
     */
    public function setMerchantCountryCode($merchantCountryCode): void
    {
        $this->merchantCountryCode = $merchantCountryCode;
    }

    /**
     * @return mixed
     */
    public function getMerchantCity()
    {
        return $this->merchantCity;
    }

    /**
     * @param mixed $merchantCity
     */
    public function setMerchantCity($merchantCity): void
    {
        $this->merchantCity = $merchantCity;
    }

    /**
     * @return mixed
     */
    public function getMerchantState()
    {
        return $this->merchantState;
    }

    /**
     * @param mixed $merchantState
     */
    public function setMerchantState($merchantState): void
    {
        $this->merchantState = $merchantState;
    }

    /**
     * @return mixed
     */
    public function getMerchantZipCode()
    {
        return $this->merchantZipCode;
    }

    /**
     * @param mixed $merchantZipCode
     */
    public function setMerchantZipCode($merchantZipCode): void
    {
        $this->merchantZipCode = $merchantZipCode;
    }

    /**
     * @return mixed
     */
    public function getMerchantDesc()
    {
        return $this->merchantDesc;
    }

    /**
     * @param mixed $merchantDesc
     */
    public function setMerchantDesc($merchantDesc): void
    {
        $this->merchantDesc = $merchantDesc;
    }


}

?>